### DISCUSSION TITLE
suggestion 2

### DISCUSSION THREAD

**Discussion by bangbangyouall** [lightbeam24/CustomTube#89] [General]
welcome to another suggestion thread!
![bro](https://github.com/lightbeam24/CustomTube/assets/109046841/c0c6db9b-99db-4dba-9d60-9d1a8179d19c)
let's start off with a bugfix idea. if you look closely you will see the problem. the player is a little off center & that is why the player gets overlapped by the video suggestions at the right a little bit. move the player 1 pixel to the left to where it wont get overlapped.

![image](https://github.com/lightbeam24/CustomTube/assets/109046841/3b619698-5b4d-415a-98b9-087d5f4702f4)
2014 also suffers from this. move the player 1 pixel to the left too.

![the hitchhiker gets all the layouts](https://github.com/lightbeam24/CustomTube/assets/109046841/c4e24b29-a55a-44ec-8e0a-10f40e7aa550)
i think it's kinda weird that the hitchhiker gets all the layouts. make 2014-2017 category nirvana (unofficial codename) to seperate it a little bit better, keep the 2013 category name hitchhiker.

**Comment by lightbeam24** [comment]
> let's start off with a bugfix idea. if you look closely you will see the problem. the player is a little off center & that is why the player gets overlapped by the video suggestions at the right a little bit. move the player 1 pixel to the left to where it wont get overlapped.

See, the problem with bugs like this is that they often depend on what zoom level, browser, or even OS you're on (due to font rendering differences). A good example of this is #5. I could look into it, but there's a fairly decent chance it's unfixable without breaking something else.

> i think it's kinda weird that the hitchhiker gets all the layouts. make 2014-2017 category nirvana (unofficial codename) to seperate it a little bit better, keep the 2013 category name hitchhiker.

Hmm, there's an idea. I'll think about it. I do think that the 2013 layouts could be considered a completely separate layout if it wasn't for the 2014 layout being a mix of 2013 and 2015. The only problem is that most people refer to the 2014-2017 layout as "Hitchhiker" not Nirvana. Heck, I didn't even know it was also called Nirvana until after the extension released. Maybe I could called it "Nirvana Hitchhiker Layout", but I could see that causing confusion. Perhaps putting the 2013 layouts in a new category called "Classic Hitchhiker Layout" would be a better idea.

**Reply by bangbangyouall** [reply]
![it dont matter what res](https://github.com/lightbeam24/CustomTube/assets/109046841/f448bf3d-e01f-4fa3-9702-14fa9df9926e)
![also on 2014](https://github.com/lightbeam24/CustomTube/assets/109046841/57af13b0-ca5a-40a8-81db-8bf58f3fa4f3)
(look closely)
the 2013 & 2014 issue happens on all zoom resolutions btw, i just zoomed in so you could see what i mean! also, i like the idea of hitchhiker classic & hitchhiker modern.

**Reply by bangbangyouall** [reply]
hello?

**Reply by lightbeam24** [reply]
What? I had nothing else to say.

**Reply by bangbangyouall** [reply]
oooohh, i thought you would say something along the lines of "ok, i will look into it."

**Reply by lightbeam24** [reply]
I think I already told you that the video player being 1 pixel off is very low priority for me. I guess I'll look into it eventually, but chances are I won't do that until early 2024.

I guess I'll say that I'm thinking of making the 2013 layouts "Classic Hitchhiker Layout" in either the next update or a couple updates after that (It's not hard to do at all, I'm just still deciding whether or not I want to do it).

**Reply by bangbangyouall** [reply]
ok, here's some more ideas:

whenever you hover over a layout, a thumbnail of it shows up on the right, just a bit of extra polish. give the user an option to disable those previews, though.

make material search bar a bit darker.

add an option to redirect a shorts video into a watch page.

make the notification icon in all 2013s a red square with the number of notifications inside & if the square is out of notifications, it is a white square, like in 2013internetlover's rehike 2013 theme.

make the early 2013 subscribe button always remain like it is in light mode.

if a layout has multiple variations, make it an option that whenever you hover over it, it is a dropdown menu.


also, i think fixing the video player being 1 pixel off in 2013s-2014 in early 2024 is a bit long, maybe at the beginning of november is right for me.

also also, yeah, i think that i like the idea of classic hitchhiker & standard hitchhiker is a good idea, maybe put 2013 & 2014 into the classic category.

**Reply by lightbeam24** [reply]
> whenever you hover over a layout, a thumbnail of it shows up on the right, just a bit of extra polish. give the user an option to disable those previews, though.

I've thought about that, and even tried to implement it at one point, but it would be a fair bit amount of work for little reward. Other things to consider would be what page is shows the screenshot of, and how big the thumbnail should be.

> make material search bar a bit darker.

The option is broken right now actually, I want to get to fixing it soon.

> add an option to redirect a shorts video into a watch page.

Not planned as of now.

> make the notification icon in all 2013s a red square with the number of notifications inside & if the square is out of notifications, it is a white square, like in 2013internetlover's rehike 2013 theme.

We'll see.

> make the early 2013 subscribe button always remain like it is in light mode.

Looks like it's bugged in dark mode. Will fix eventually, but it's a pretty minor issue.

> if a layout has multiple variations, make it an option that whenever you hover over it, it is a dropdown menu.

???

If you mean to make stuff like the 2013 layouts selectable in a dropdown, then no, this is not planned.

> maybe put 2013 & 2014 into the classic category.

This is why I'm not so sure about this idea. What determines the difference between classic and modern hitchhiker? Does Nirvana define modern? Or does Watch8 define modern? That's what ultimately made me put them all in the same category in the first place.

> also, i think fixing the video player being 1 pixel off in 2013s-2014 in early 2024 is a bit long, maybe at the beginning of november is right for me.

Then do it yourself. Make a userstyle that fixes it, I have no obligation to fix this "issue". Like seriously, I don't even know what would be causing this, and looking for it would be a ridiculous waste of time for me.

I need you to understand something: **CustomTube is an extension I made for myself to use. I'm simply letting others use it as well.** It's also just something I make for fun, it's not my job. This means that what *I* want to be added/changed comes before what anyone else wants. This 1 pixel bug in the 2013-2014 layouts is **meaningless** to me. I do take suggestions sometimes, but literally no one else has asked for this, or probably even noticed.

If you're going to keep pushing for this, I'm going to start ignoring your future suggestions. I have my own plan for this extension, and it does not align with yours at all.
